mockintosh/handlers.py

```diff
@@ -90,9 +115,11 @@ def super_verb(self, *args):
             self.dynamic_unimplemented_method_guard()
 
         try:
-            _id, response, params, context, dataset = self.match_alternative()
+            _id, response, params, context, dataset, internal_endpoint_id = self.match_alternative()
         except TypeError:
```

Comment:
Why is this exception eaten up? Shouldn't we at least log it?

Reply:
Fixed with https://github.com/up9inc/mockintosh/commit/02fdb183d282b06f0fe14df0dce54d6df77bb4c9